app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/twofactorlogin/TwoFactorLoginScreen.kt

```diff
@@ -295,3 +297,31 @@ private fun TwoFactorLoginScreenContent(
         }
     }
 }
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+@Preview(showBackground = true)
+private fun TwoFactorLoginScreenContentPreview() {
+    BitwardenTheme {
+        BitwardenScaffold {
```

Comment:
Do we need the scaffold?

Reply:
It isn't _needed_, but it makes the preview render more closely to how it will appear on a device.